tests/behaviour/background/cluster/environment.py

```diff
@@ -42,6 +42,9 @@ def before_scenario(context: Context, scenario):
             scenario.skip("tagged with @" + tag)
             return
     environment_base.before_scenario(context, scenario)
+    context.session_options = TypeDBOptions.cluster().set_infer(True)
+    context.transaction_options = TypeDBOptions.cluster().set_infer(True)
```

Comment:
we now have session and transaction options we have configure before each scenario